src/pages/Product-Details/Product-details.css

```diff
@@ -0,0 +1,86 @@
+.product-details-main {
+  margin: auto;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  min-height: 88vh;
+}
+
+.product-details {
+  min-height: 70vh;
+  width: 80%;
+  display: flex;
+  padding: 1.2rem;
+}
+
+.product-details div:first-child {
+  flex: 0 1 50%;
+  background-color: var(--bg-darkWhite);
+  padding: 2.4rem 4.8rem;
+  position: relative;
+}
+
+.product-details div:first-child img {
+  border-radius: 8px;
+}
+
+.product-details .content {
+  flex: 0 0 50%;
+  padding: 0 4.8rem;
+}
+
+.product-name {
+  color: #000;
```

Comment:
A suggestion : if feasible , for future iterations use variables for color , instead hard coding.